code/modules/cm_preds/yaut_weapons.dm

```diff
@@ -1396,6 +1396,203 @@
 		user.update_power_display(perc)
 	return TRUE
 
+/obj/item/weapon/gun/bow
+	name = "hunting bow"
+	desc = "An abnormal-sized weapon with an exeptionally tight string. Requires extraordinary strength to draw."
+	icon = 'icons/obj/items/hunter/pred_gear.dmi'
+	icon_state = "bow"
+	item_state = "bow"
+	item_icons = list(
+		WEAR_L_HAND = 'icons/mob/humans/onmob/hunter/items_lefthand.dmi',
+		WEAR_R_HAND = 'icons/mob/humans/onmob/hunter/items_righthand.dmi',
+		WEAR_BACK = 'icons/mob/humans/onmob/hunter/pred_gear.dmi'
+	)
+	current_mag = /obj/item/ammo_magazine/internal/bow
+	reload_sound = 'sound/weapons/gun_shotgun_shell_insert.ogg'
+	fire_sound = 'sound/weapons/bow_shot.ogg'
+	aim_slowdown = 0
+	flags_equip_slot = SLOT_BACK
+	flags_gun_features = GUN_INTERNAL_MAG|GUN_CAN_POINTBLANK|GUN_WIELDED_FIRING_ONLY|GUN_UNUSUAL_DESIGN
+	gun_category = GUN_CATEGORY_HEAVY
+	muzzle_flash = null
+	w_class = SIZE_LARGE
+	explo_proof = TRUE
+	unacidable = TRUE
+	flags_item = TWOHANDED|ITEM_PREDATOR
+
+/obj/item/weapon/gun/bow/Initialize(mapload, spawn_empty)
+	. = ..(mapload, TRUE) //is there a better way?
```

Comment:
you could do this instead if you prefer this syntax

```
/obj/.../Initialize(mapload, spawn_empty)
  spawn_empty = TRUE
  . = ..()
  update_icon()
  ```
  
  are you sure you need update_icon ? it's already done in parent